src/main/java/net/imglib2/RealRandomAccessible.java

```diff
@@ -48,6 +48,10 @@
  * expect that the domain is infinite. (In contrast to this,
  * {@link RealRandomAccessibleRealInterval}s have a finite domain.)
  * </p>
+ * <p>
```

Comment:
That one I guessed, based on the statement about `RandomAccessible` not being defined everywhere.